corehq/apps/users/views/mobile/users.py

```diff
@@ -1148,8 +1148,14 @@ def post(self, request, *args, **kwargs):
 
         user_docs_by_id = {doc['_id']: doc for doc in get_user_docs_by_username(usernames)}
         user_ids_with_forms = self._get_user_ids_with_forms(request, user_docs_by_id)
-        self._handle_usernames_not_found(request, user_docs_by_id, usernames)
-        self._delete_users(request, user_docs_by_id, user_ids_with_forms)
+        usernames_not_found = self._get_usernames_not_found(request, user_docs_by_id, usernames)
```

Comment:
nit: should that be renamed to `validate_usernames` or something similar
`users_not_found = validate_usernames(request, user_docs_by_id, usernames)`

Reply:
Going to leave as is; it's less clear what `validate_usernames` would validate and what it would return.